addon/helpers/sort-by.js

```diff
@@ -0,0 +1,192 @@
+// taken from Ember Composable Helpers (https://github.com/DockYard/ember-composable-helpers), then modified.
```

Comment:
Wondering if we can import the sorter from https://github.com/ilios/common/blob/master/addon/utils/array-helpers.js#L28 and use that here? Maybe another step.

Reply:
good idea, but should be a follow-up item. will create a corresponding ticket.